app/src/main/java/com/teamltt/carcare/database/contract/ReminderContract.java

```diff
@@ -0,0 +1,140 @@
+/*
+ *
+ *  * Copyright 2017, Team LTT
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.teamltt.carcare.database.contract;
+
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.teamltt.carcare.model.Reminder;
+
+public class ReminderContract {
+
+    public static final String SQL_CREATE_ENTRIES = "CREATE TABLE " + ReminderEntry.TABLE_NAME + " ("
+            // vehicle_id INTEGER PRIMARY KEY
+            + ReminderEntry.COLUMN_NAME_ID + " INTEGER PRIMARY KEY AUTOINCREMENT,"
+            + ReminderEntry.COLUMN_NAME_VEHICLE_ID + " INTEGER,"
+            + ReminderEntry.COLUMN_NAME_NAME + " STRING,"
+            + ReminderEntry.COLUMN_NAME_FEATURE_ID + " INTEGER,"
+            + ReminderEntry.COLUMN_NAME_COMPARISON + " INTEGER,"
+            + ReminderEntry.COLUMN_NAME_VALUE + " INTEGER,"
+            + ReminderEntry.COLUMN_NAME_DATE + " DATE"
+            + ");";
+
+    public static final String SQL_DROP_ENTRIES = "DROP TABLE IF EXISTS " + ReminderContract.ReminderEntry.TABLE_NAME;
+
+    //get one specific reminder corresponding to reminderId
+    public static Cursor queryByReminderId(SQLiteDatabase db, long reminderId) {
+        String table = ReminderEntry.TABLE_NAME;
+        String[] columns = {
+                ReminderEntry.COLUMN_NAME_ID,
+                ReminderEntry.COLUMN_NAME_VEHICLE_ID,
+                ReminderEntry.COLUMN_NAME_NAME,
+                ReminderEntry.COLUMN_NAME_FEATURE_ID,
+                ReminderEntry.COLUMN_NAME_COMPARISON,
+                ReminderEntry.COLUMN_NAME_VALUE,
+                ReminderEntry.COLUMN_NAME_DATE
+        };
+        String selection = ReminderEntry.COLUMN_NAME_ID + " = ?";
+        String[] selectionArgs = {Long.toString(reminderId)};
+
+        return db.query(table, columns, selection, selectionArgs, null, null, null);
+    }
+
+    //get all the reminders owned by a specific vehicle
+    public static Cursor queryByVehicleId(SQLiteDatabase db, long vehicleId) {
+        String table = ReminderEntry.TABLE_NAME;
+        String[] columns = {
+                ReminderEntry.COLUMN_NAME_ID,
+                ReminderEntry.COLUMN_NAME_VEHICLE_ID,
```

Comment:
If `vehicleId` is passed in, it shouldn't be necessary to include in the returned columns.